changelog.json

```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.6.3-beta3": [
+      "[Fixed] Remote git operations (like cloning a repo) won't fail on old macOS versions - #11516",
+      "[Fixed] Updated about modal dialog on macOS to match the new Big Sur icon - #11511"
```

Comment:
I added this here since it's a fix from the previous beta, but we shouldn't add this to the prod release. I'm happy to remove it from here if needed.

Reply:
Nope, this is perfect, thank you!